### ISSUE TITLE
The training process has become stuck

### ISSUE THREAD

**Issue by qybing** [bmaltais/kohya_ss#1626]
`checking cache validity...
100%|█████████████████████████████████████████████████████████████████████████████████| 76/76 [00:00<00:00, 610.67it/s]
caching latents...
0it [00:00, ?it/s]
create LoRA network. base dim (rank): 8, alpha: 1.0
neuron dropout: p=None, rank dropout: p=None, module dropout: p=None
create LoRA for Text Encoder 1:
create LoRA for Text Encoder 2:
create LoRA for Text Encoder: 264 modules.
create LoRA for U-Net: 722 modules.
enable LoRA for text encoder
enable LoRA for U-Net
prepare optimizer, data loader etc.
use Adafactor optimizer | {'scale_parameter': False, 'relative_step': False, 'warmup_init': False}
because max_grad_norm is set, clip_grad_norm is enabled. consider set to 0 / max_grad_normが設定されているためclip_grad_normが有効になります。0に設定して無効にしたほうがいいかもしれません
constant_with_warmup will be good / スケジューラはconstant_with_warmupが良いかもしれません
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 760
  num reg images / 正則化画像の数: 0
  num batches per epoch / 1epochのバッチ数: 760
  num epochs / epoch数: 2
  batch size per device / バッチサイズ: 1
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 1520

`
**I have been stuck in this place, how should I solve it?**
